docs/topics_5/web-analyze-apps.adoc

```diff
@@ -3,17 +3,11 @@
 [id='using_web_console_analyze_apps_{context}']
 = Analyzing applications with the {WebName}
 
-In order to use the {WebName} to analyze applications, you must create a project. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
+To use the {WebName} to analyze applications, you must create one or more projects. Each project is a way to group applications for analysis and specifies the settings to use during the analysis. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
```

Comment:
Didn't I mark this with changes in one of my other comments?

Reply:
I moved the relevant part to master, and commented out the file, but did not remove the file yet. I wanted to wait until you saw the preview version in which it was removed.